requirements-dev.txt

```diff
@@ -4,3 +4,7 @@ flake8
 tox
 bumpversion
 twine
+arrow
```

Comment:
This file is meant to include the dependencies that are only necessary for development. This way there is no duplicated deps to keep in sync. You can run `pip install -e .` in the project root to install the runtime dependencies in your virtualenv.